src/EnergyPlus/ZoneTempPredictorCorrector.hh

```diff
@@ -226,6 +247,12 @@ namespace ZoneTempPredictorCorrector {
 	CalcZoneAirTempSetPoints();
 
 	void
+	CalculateMonthlyRunningAverageDryBulb( Array1D< Real64 > & runningAverageASH, Array1D< Real64 > & runningAverageCEN );
```

Comment:
@EnergyArchmage @mbadams5 Thanks for the clarification. Lesson learned. I've changed it in the latest commit.